src/main/python/apache/thermos/core/process.py

```diff
@@ -348,19 +349,31 @@ def execute(self):
     if os.path.exists(thermos_profile):
       env.update(BASH_ENV=thermos_profile)
 
-    self._popen = subprocess.Popen(["/bin/bash", "-c", self.cmdline()],
-                                   stderr=self._stderr,
-                                   stdout=self._stdout,
-                                   close_fds=self.FD_CLOEXEC,
-                                   cwd=sandbox,
-                                   env=env)
+    subprocess_args = {
+      'args': ["/bin/bash", "-c", self.cmdline()],
+      'close_fds': self.FD_CLOEXEC,
+      'cwd': sandbox,
+      'env': env,
+      'pathspec': self._pathspec
+    }
+
+    if self._log_maxbackups > 0 and self._log_maxbytes > 0:
```

Comment:
Should self._log_maxbackups default to 1 if self._log_maxbytes > 0 and self._log_maxbackups == 0?